src/agent_factory/instructions.py

```diff
@@ -156,7 +160,7 @@ def run_agent(url: str):
     and return structured output.
     \"\"\"
     input_prompt = f"Translate the main text content from the following English webpage URL to Italian: {url}"
-    agent_trace = agent.run(prompt=input_prompt)
+    agent_trace = agent.run(prompt=input_prompt, max_turns=20)
```

Comment:
Is `max_turns` necessary for this example? What's the reasoning behind providing this?

Reply:
Yes, I was worried for the extremes: what happens if it sets it to `1` or `1000`. But I think it will just copy it. Maybe we can make a note in the README.